MekHQ/src/mekhq/campaign/mission/BotForce.java

```diff
@@ -231,7 +231,11 @@ public void writeToXml(PrintWriter pw1, int indent) {
         if (!getCamouflage().hasDefaultFilename()) {
             MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "camoFileName", getCamouflage().getFilename());
         }
-        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "colour", getColour().name());
+        
+        if (getColour() != null) {
+                MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "colour", getColour().name());
+        }
```

Comment:
Reopened #2503, as this is not a bugfix but rather an illegal statement. PlayerColour is by design non-nullable, and if it is null that will cause deeper issues.

Reply:
Sure, and if you want to track down what causes the color to be set to null, you're welcome to. I'll make it write out some default color, but that's about the limit of my interest in this problem.

Reply:
Given not a single report of this in stable, I consider this to be a bug in StratCon.

Reply:
Fair enough, I've adjusted the no-parameters constructor, which is the only place where the color doesn't get initialized.